Luminis.AzureStorageBackup/BackupAzureStorage.cs

```diff
@@ -90,9 +90,11 @@ public async Task BackupAzureTablesToBlobStorage(IEnumerable<string> sourceTable
 
             this.logger?.LogInformation($"BackupAzureTablesToBlobStorage - From: {this.sourceAccountName}, Tables: {string.Join(", ", sourceTables)} to {targetAccountName}/{targetContainerName}");
 
+            string journalFileFolder = $@"{Environment.GetEnvironmentVariable("LocalAppData")}\{Guid.NewGuid()}";
```

Comment:
CCan you put this in a temp folder instead of localAppData? That way, clean-up should be easier.